examples/src/main/java/ai/djl/examples/training/TrainSentimentAnalysis.java

```diff
@@ -199,9 +199,7 @@ public static StanfordMovieReview getDataset(
                                                 (m) -> m.ones(new Shape(1)).mul(paddingTokenValue))
                                         .build())
                         .setSourceConfiguration(
-                                new TextData.Configuration()
-                                        .setTextEmbedding(new ModelZooTextEmbedding(embeddingModel))
-                                        .setTextProcessors(TEXT_PROCESSORS))
+                                new TextData.Configuration().setTextProcessors(TEXT_PROCESSORS))
```

Comment:
How does remove `new ModelZooTextEmbedding(embeddingModel)` solve the issue

Reply:
`ModelZooTextEmbedding` is used as a part of the model. Now we apply a duplicate layer on the top. This example never work from 0.6.0.